documentation.md

```diff
@@ -39,25 +39,26 @@ You may find a quick overview about the [main features](/features.html) useful.
 ### API Documentation
 We provide several APIs to interact with SirixDB.
 
-1. The [transactional cursor API](/transactional-cursor-api.html) is a powerful low-level API.
-2. On top of this API we built a [Brackit.org](http://brackit.org) binding to provide the ability to use SirixDB with a more [DOM-alike API](/dom-alike-api.html) with in-memory nodes and an [XQuery API](/xquery-api.html).
-3. We provide a powerful, asynchronous, non-blocking [RESTful-API](/rest-api.html) to interact with a SirixDB HTTP-server. Authorization is done via Keycloak.
+1. The transactional cursor API is a powerful low-level API.
+2. On top of this API, we built a [Brackit.org](http://brackit.org) binding to provide the ability to use SirixDB with a more [DOM-alike API](/dom-alike-api.html) with in-memory nodes and an [XQuery API](/xquery-api.html).
+3. We provide a powerful, asynchronous, non-blocking [RESTful-API](/rest-api.html) to interact with a SirixDB HTTP server. Authorization is done via Keycloak.
 
 ### Publications
 Articles published on Baeldung:
 - [A Guide to SirixDB](https://www.baeldung.com/sirix)
 
 Articles published on Medium: 
-- [Asynchronous, Temporal  REST With Vert.x, Keycloak and Kotlin](https://medium.com/hackernoon/asynchronous-temporal-rest-with-vert-x-keycloak-and-kotlin-coroutines-217b25756314)
-- [Pushing Database Versioning to Its Limits by Means of a Novel Sliding Snapshot Algorithm and Efficient Time Travel Queries](https://medium.com/sirixdb-sirix-io-how-we-built-a-novel-temporal/why-and-how-we-built-a-temporal-database-system-called-sirixdb-open-source-from-scratch-a7446f56f201)
+- [Asynchronous, Temporal REST With Vert.x, Keycloak, and Kotlin](https://medium.com/hackernoon/asynchronous-temporal-rest-with-vert-x-keycloak-and-kotlin-coroutines-217b25756314)
+- [Pushing Database Versioning to Its Limits using a Novel Sliding Snapshot Algorithm and Efficient Time Travel Queries](https://medium.com/sirixdb-sirix-io-how-we-built-a-novel-temporal/why-and-how-we-built-a-temporal-database-system-called-sirixdb-open-source-from-scratch-a7446f56f201)
```

Comment:
Please don't change the titles of the articles, this is title under which the article was published.